portfolio-styling.css

```diff
@@ -1,81 +1,181 @@
-/* Header, footer and nav */
+/* Basic stuff for reset, site styles and text sizing */
 
 * {
 	box-sizing: border-box;
 
 }
 
-h1, h2, h3, p, img, figure {
+h1, h2, h3, h4, h5, h6, p, a {
 	margin: 0;
-	border: 0;
 	padding: 0;
+	text-decoration: none;
+	font-weight: normal;
 }
 
-header.portfolio-site-header {
-	display: flex;
-	flex-flow: row wrap;
-	justify-content: flex-start;
-	align-items: baseline;
-	position: sticky; /* keep element where it is without removing from document flow */
-	top: 0; /* set a distance from the top to stick to - 0 means always stay at the top */
-	background-color: rgb(75, 75, 75);
+html {
+	font-size: 15px;
+	font-family: "Courier New";
 }
 
-footer.portfolio-site-footer {
-	position: sticky;
-	bottom: 0;
-	background-color: rgb(75, 75, 75);
+h1 {
+	white-space: nowrap;
+	display: none;
 }
 
+h2 {
+	font-size: 2.4 rem;
+	width: 100%;
+	text-align: center;
+}
+
+h2.section-title {
+	visibility: hidden;
+	margin: 20px 0 20px 0;
+}
+
+h3 {
+	font-size: 1.8 rem;
+	width: 100%;
+	margin: 5% 0 5% 0;
+	text-align: center;
+}
+
+p {
+	font-size: 1.2 rem;
+	padding: 8%;
+}
 
-h1.portfolio-title	{
+a {
+	padding: 2px;
 	white-space: nowrap;
+	font-weight: 500;
 }
 
-nav.portfolio-site-nav {
-	display: flex;
-	flex-flow: row nowrap;
-	justify-content: space-evenly;
-	flex-grow: 2;
+a.cheat-sheet-link {
+	text-decoration: underline;
+}
+
+a.portfolio-nav-link {
+	font-size: 1.4rem ;
 }
 
 
-/* MAIN CONTENT */
+/* Set default backgrounds and text colours  */
 
-h2, h3 {
-	width: 100%;
-	margin: 10px 0 10px 0;
-	text-align: center;
+body, header, footer {
+	background-color: rgb(0, 0, 0);
 }
 
-p {
+h1, h3, p, a {
+	color: rgb(180, 180, 180);
+}
 
+figcaption {
+	text-decoration: none;
+	color: rgb(102, 102, 102);
 }
 
+
+/* Styling for all elements (no custom classes) */
+
 section {
-	margin: 5% 0 5% 0;
 	display: flex;
 	flex-flow: row wrap;
 	justify-content: center;
 }
 
+nav {
+	padding: 10px;
+	border-bottom: 1px solid rgb(180, 180, 180);;
+}
+
 article {
 	min-width: 280px;
-	max-width: 40%;
-	margin: 5% auto;
+	max-width: 400px;
+	margin: 10px;
 	border: 1px solid black;
+	border-radius: 25px;
+	background-color: rgb(51, 51, 51);
+	display: flex;
+	flex-flow: row wrap;
 }
 
 figure {
 	max-width: 80%;
 	margin: 0 auto;
+}
 
+figcaption {
+	text-align: center;
 }
 
-img.about-image, img.project-image {
-	max-width: 100%;
+iframe {
+	width: 100%;
+	height: 400px;
+}
+
+footer {
 	margin: 0 auto;
+	padding: 5px 0 10px 0;
+	width: 30%;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	border-radius: 100px;
+}
+
+
+/* Styling for custom classes ============================== */
+
+section.about-section {
```

Comment:
you can add the article.about-content selector with a "," since they have the same property and value, to be more explicit in case I don't make myself clear: `selector-1, selector-2 {property:value;}`